source/wp-content/themes/wporg-developer-blog/style.css

```diff
@@ -14,3 +14,85 @@
  * Note: only add styles here in cases where you can't achieve the style with
  * templates or theme.json settings.
  */
+
+/*
+ * Add inline styling for code and kbd markup.
+ */
+:where( :not( pre ) > code ),
+kbd {
+    background: var(--wp--preset--color--light-grey-2);
+    border-radius: 2px;
+    display: inline-block;
+    line-height: var(--wp--custom--body--extra-small--typography--line-height);
+    max-width: 100%;
+    padding-left: 6px;
+    padding-right: 6px;
+}
+
+/*
+ * Remove text-decoration from linked footer headings.
+ */
+.wporg-front-page-footer .wp-block-heading a {
+    text-decoration: none
+}
+
+.wporg-front-page-footer .wp-block-heading a:hover {
+    text-decoration-line: underline;
+}
+
+/*
+ * Fix local navigation submenu styles.
+ */
+.wp-block-navigation ul {
+    padding: 0 !important;
+}
+
+.wp-block-navigation .wp-block-navigation__submenu-container {
+    border: 0 !important;
+    background-color: #1c2024 !important;
```

Comment:
Where does this color come from?

Reply:
This comes from the variables defined in the header. The [`--wp--preset--color--nero` value](https://github.com/WordPress/wporg-mu-plugins/blob/dd5cd128ff5db670a417511d5c9a5babaabbed65/mu-plugins/blocks/global-header-footer/postcss/_common.pcss#L74) is not available in parent theme for some reason, so I just hardcoded the value. 

<img width="400" alt="image" src="https://github.com/WordPress/wporg-developer-blog/assets/4832319/68e1cff4-d1c3-45b1-a4c3-78b67731ecb5">